packages/components/src/components/samples/SampleSetParentAliasRow.tsx

```diff
@@ -58,29 +58,41 @@ export class SampleSetParentAliasRow extends React.Component<IParentAliasRow> {
     };
 
     onSelectChange = (name: string, selectedValue: string, selectedOption: IParentOption): void => {
-        this.props.onAliasChange(this.props.id, name, selectedOption);
+        // Issue 40149: on clear, need to retain the IParentOption with schema
+        let newValue = selectedOption;
+        if (!selectedOption) {
+            newValue = { schema: this.props.parentAlias.parentValue.schema };
```

Comment:
Do we know that parentValue is defined at this point?

Reply:
Good point. I'll add a check.